app/controllers/hyrax/pages_controller.rb

```diff
@@ -7,7 +7,7 @@ class PagesController < ApplicationController
     helper Hyrax::ContentBlockHelper
 
     def show
-      @page = ContentBlock.find_or_create_by(name: params[:id])
+      @page = ContentBlock.public_send(params[:id].to_sym)
```

Comment:
Use the getter method on `ContentBlock` to initialize missing blocks. This allows each block to initialize in its own way; `terms_page` and `agreement_page` have default values, and line 10 previously bypassed the code that ensured default values were used upon initialization.